packages/css-library/config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "source": ["tokens/**/*.json"],
+  "platforms": {
+    "scss": {
+      "transformGroup": "scss",
+      "buildPath": "src/tokens/",
+      "files": [{
+        "destination": "_variables.scss",
```

Comment:
We can name the filename and path to whatever we want and make sure that those files are included in the npm package.